framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -973,21 +974,44 @@ public void setTypeArguments(List<? extends AnnotatedTypeMirror> ts) {
     }
 
     /**
-     * Returns true if the type was raw, that is, type arguments were not provided but instead
-     * inferred.
+     * Returns true if the underlying type is raw. The receiver of this method is not raw, however;
+     * its annotated type arguments have been inferred.
      *
      * @return true iff the type was raw
      */
+    public boolean isUnderlyingTypeRaw() {
+      return isUnderlyingTypeRaw;
+    }
+
+    /**
+     * Returns true if the underlying type is raw. The receiver of this method is not raw, however;
+     * its annotated type arguments have been inferred.
+     *
+     * @return true iff the type was raw
+     * @deprecated Use {@link #isUnderlyingTypeRaw()} instead
+     */
+    @Deprecated
```

Comment:
Please add a date to each `@Deprecated` annotation to help us know when to remove the deprecated methods.  (I did so for this pull request.)